prow/sidecar/censor_test.go

```diff
@@ -122,6 +122,14 @@ func TestCensorIntegration(t *testing.T) {
 	}); err != nil {
 		t.Fatalf("failed to copy input to temp dir: %v", err)
 	}
+	// also, tar the input - it's not trivial to diff two tarballs while only caring about
+	// file content, not metadata, so this test will tar up the archive from the input and
+	// untar it after the fact for simple diffs and updates
+	archiveDir := filepath.Join(tempDir, "artifacts/archive")
```

Comment:
Can we shell out to `tar` for this so we can be confident that our untargzip/targzip implementation is correct?

Reply:
Yeah, that was a thought.